README.md

```diff
@@ -1,3 +1,41 @@
-# harness-operator
+# infra-operator
 
-th2 cluster operator
\ No newline at end of file
+infra-operator is java implementation of Kubernetes 
+[custom resource controller](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-controllers). 
+It is part of _th2 infrastructure_. Together with [infra-mgr](https://github.com/th2-net/th2-infra-mgr) and 
+[helm-operator](https://github.com/fluxcd/helm-operator) 
+it ensures synchronization of custom resource files from GitHub and actual resources 
+in Kubernetes. infra-operator uses [fabric8](https://fabric8.io/guide/) library for communication with Kubernetes. 
+
+[Custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) 
+allow us to extend Kubernetes API with custom components specifically designed for our needs. 
+However, as such custom components are not part of default Kubernetes installation It is infra-operator's 
```

Comment:
However, **since** such custom components are not part of **the** default Kubernetes installation **, it is** infra-operator's